src/test/java/com/commercetools/sync/sdk2/categories/CategorySyncMockUtils.java

```diff
@@ -0,0 +1,246 @@
+package com.commercetools.sync.sdk2.categories;
```

Comment:
This can be pulled from the base [java-sdk-v2-category-sync-migration](https://github.com/commercetools/commercetools-sync-java/tree/java-sdk-v2-category-sync-migration)